Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -155,13 +155,14 @@ Use ``--verbose`` for full details:
     $ kubectl exec -it -n kube-system cilium-fmh8d -- cilium status --verbose
     [...]
     KubeProxyReplacement Details:
-      Status:              Strict
-      Protocols:           TCP, UDP
-      Devices:             eth0 (Direct Routing), eth1
-      Mode:                SNAT
-      Backend Selection:   Random
-      Session Affinity:    Enabled
-      XDP Acceleration:    Disabled
+      Status:                Strict
+      Protocols:             TCP, UDP
+      Devices:               eth0 (Direct Routing), eth1
+      Mode:                  SNAT
+      Backend Selection:     Random
+      Session Affinity:      Enabled
+      Graceful Termination:  Enabled
```

Comment:
Should there also be a sentence in the graceful termination section that when the detection is enabled, `cilium status --verbose` will show it.